README

```diff
@@ -1 +1,67 @@
-Presto Manager: A tool for managing Presto clusters.
+# Presto Manager
+
+A tool for managing Presto clusters.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+Java 8, available at (http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+Maven 3+, available at (https://maven.apache.org/download.cgi)
+Python 2.4+, available at (https://www.python.org/download/releases/2.4/)
+
+### Installing
+
+Clone the full distribution from (https://github.com/prestodb/presto-manager)
+
+```
+git clone https://github.com/prestodb/presto-manager.git
+```
+Go the the directory you cloned the distribution
+
+```
+cd /directory/of/the/clone/presto-manager
```

Comment:
`cd presto-manager`; the cloned repo will be in the same folder you are in. Actually, this step probably belongs in "Compiking and Running".